src/cli/usb.js

```diff
@@ -0,0 +1,109 @@
+const settings = require('../../settings');
+
+function usbCommand() {
+	if (!usbCommand._instance) {
+		const UsbCommand = require('../cmd/usb');
+		usbCommand._instance = new UsbCommand(settings);
+	}
+	return usbCommand._instance;
+}
+
+module.exports = ({ commandProcessor, root }) => {
+	const usb = commandProcessor.createCategory(root, 'usb', 'Control USB devices');
+
+	commandProcessor.createCommand(usb, 'list', 'List the devices connected to the host computer', {
+		options: {
+			'exclude-dfu': {
+				description: 'Do not list devices which are in DFU mode',
+				boolean: true
+			},
+			'ids-only': {
+				description: 'Print only device IDs',
+				boolean: true
+			}
+		},
+		handler: (args) => {
+			return usbCommand().list(args);
+		}
+	});
+
+	// Common options for start-listening, stop-listening, safe-mode, dfu and reset
+	const commonOptions = {
+		'all': {
+			description: 'Send the command to all devices connected to the host computer',
+			boolean: true
+		},
+		'one': {
+			description: 'Send the command to a single device connected to the host computer',
```

Comment:
In other CLI commands, this mode is the default. If you do `particle flash --usb` it just picks the one device connected. I'd like to avoid the need to specify a device ID if there is only one device connected.